datagen/datagen_utils.py

```diff
@@ -45,11 +45,44 @@ def filter_pickupable(
     ]
 
 
+def get_random_seeds(max_seed: int = int(1e8)) -> Dict[str, int]:
+    # Generate random seeds for each stage
+
+    # Train seed
+    random.seed(1329328939)
+    train_seed = random.randint(0, max_seed - 1)
+
+    # Train unseen seed
+    random.seed(709384928)
+    train_unseen_seed = random.randint(0, max_seed - 1)
+
+    # val seed
+    random.seed(3348958620)
+    val_seed = random.randint(0, max_seed - 1)
+
+    # test seed
+    random.seed(289123396)
+    test_seed = random.randint(0, max_seed - 1)
```

Comment:
maybe remove this 😄

Reply:
We don't need to since the test dataset is public anyway :)